src/main/java/de/picturesafe/search/elasticsearch/config/impl/StandardFieldConfiguration.java

```diff
@@ -182,6 +201,11 @@ public Builder analyzer(String analyzer) {
             return this;
         }
 
+        public Builder withoutIndexing(boolean withoutIndexing) {
```

Comment:
You could remove the boolean method parameter, because when calling this method the intention is to set it to true.

Reply:
ok!